queries/earth.jinja2

```diff
@@ -1,78 +1,99 @@
 {% if zoom < 9 %}
 
 SELECT
-    NULL AS name,
-    'earth' AS kind,
-    NULL AS label_placement,
+    '' AS name,
+    sum(way_area)::bigint AS area,
+    mz_calculate_output_earth(t.*) AS mz_properties,
     'naturalearthdata.com' AS source,
     {% filter geometry %}st_union(the_geom){% endfilter %} AS __geometry__,
-    max(gid) AS __id__
+    max(gid) AS __id__,
+    NULL AS label_placement,
+    NULL AS boundary
 FROM
-
 {% if zoom < 2 %}
-    ne_110m_land
+    ne_110m_land t
 {% elif 2 <= zoom < 5 %}
-    ne_50m_land
+    ne_50m_land t
 {% elif 5 <= zoom < 9 %}
-    ne_10m_land
+    ne_10m_land t
 {% endif %}
 
 WHERE
+    mz_earth_min_zoom <= {{ zoom }} AND
     {{ bounds|bbox_filter('the_geom') }}
+GROUP BY mz_properties
 
-{% else %}
+{% elif 9 <= zoom %}
```

Comment:
Did you mean `zoom >= 9`? If so, would it be easier to use `else`? Apologies if not - I'm having trouble following the `if - elsif - endif` stuff through the diff!